client/account-recovery/lost-password/lost-password-form/index.jsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import support from 'lib/url/support';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+
+export class LostPasswordFormComponent extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			isSubmitting: false,
+			userLogin: '',
+		};
+	}
+
+	onSubmit = () => {
+		this.setState( { isSubmitting: true } );
+
+		//TODO: dispatch an event with userLogin and wait to here back
+	};
```

Comment:
Fun fact!

You don't need the curlies here. If you leave them out then the fat-arrow becomes a function which returns the value of the expression. In this case I think it's `null` but it's whatever `this.setState()` actually returns…

```js
onSubmit = () => this.setState( { isSubmitting: true } )
```

P.S. there might be a slightly better name available which holds more meaning than `onSubmit`, such as `submitForm`. It's somewhat murky here but in general we like to avoid `onDasher` `onDancer` `onComet` and so on for handler names.